docs/visuals/ci-reviewer-dashboard.html

```
@@ -0,0 +1,601 @@
+          </div>
+          <div class="stat">
+            <div class="label">Model freedom</div>
+            <div class="value">High</div>
+            <div class="note">The workflow uses <code>--approval-mode=yolo</code> so the agent can explore the workspace directly.</div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <div class="nav" role="tablist" aria-label="CI reviewer sections">
+      <button class="active" data-target="cadence">1. Activation cadence</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)